xsd/siri_model/siri_situationActions-v2.0.xsd

```diff
@@ -467,4 +514,252 @@ Rail transport, Roads and road transport
    </xsd:extension>
   </xsd:complexContent>
  </xsd:complexType>
+ <!-- ===Extentions for SIRI-SX v2.0p-a0.x================================================== -->
+ <xsd:complexType name="ActionPeriodStructure">
+  <xsd:sequence>
+   <xsd:element name="StartTime" type="xsd:time"/>
+   <xsd:element name="EndTime" type="xsd:time"/>
+  </xsd:sequence>
+ </xsd:complexType>
+ <xsd:complexType name="AffectsSegmentStructure">
+  <xsd:sequence>
+   <xsd:element name="Start" type="AffectedPlaceStructure"/>
+   <xsd:element name="End" type="AffectedPlaceStructure"/>
+  </xsd:sequence>
+ </xsd:complexType>
+ <xsd:complexType name="DescriptionContentStructure">
+  <xsd:sequence>
+   <xsd:element name="DescriptionText" type="DefaultedTextStructure" maxOccurs="unbounded">
+    <xsd:annotation>
+     <xsd:documentation>Description of SITUATION. Should not repeat any strap line included in Summary.  (Unbounded since SIRI 2.0)</xsd:documentation>
+    </xsd:annotation>
+   </xsd:element>
+   <xsd:element name="DescriptionPriority" type="xsd:positiveInteger" minOccurs="0">
```

Comment:
Are the xxxPriority elements mandatory if there is more than one xxxContent - or can they be optional even then?

Reply:
In the UmS group documentation it seems to be intentially optional. The UmS group documented that when priority is left out, end systems can decide how to sort/prioritise the text contents on its own. This seems reasonable for me.